guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,16 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling rh-cloud and insights-client reports on hosts
+
+You can set the `host_registration_insights` parameter to *True* to enable _rh-cloud_ and _insights-client_ reports.
+{Project} will create _rh-cloud_ reports and _insights-client_ will upload reports to the cloud.
+
+Use the following procedure to change the value of `host_registration_insights` parameter.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Host* > *Provisioning Setup* > *Operating Systems*.
+. Select any {RHEL} operating systems for which you want to change the value.
+. On the *Parameters* tab, click on the edit button of `host_registration_insights`.
+. Set the value to *True*.
+
+You can also set this parameter at the organization, host group, subnet, and domain level.
+However, if you set the parameter non any non-{RHEL} operating systems, it automatically uploads new reports.
```

Comment:
Perhaps add a note about the implications (which I don't know).

Reply:
Fix: However, if you set the parameter on any non-{RHEL} operating system, it automatically uploads new reports.